src/ViewItem.js

```diff
@@ -844,286 +978,364 @@ class ViewItem extends React.Component {
+                    <Col
+                      smOffset={0}
+                      sm={4}
+                    >
                       <KeyValue
                         label={<FormattedMessage id="ui-inventory.permanentLocation" />}
-                        value={get(permanentHoldingsLocation, ['name'], '')}
+                        value={checkIfElementIsEmpty(holdingLocation.permanentLocation)}
                       />
                     </Col>
                     <Col sm={4}>
                       <KeyValue
                         label={<FormattedMessage id="ui-inventory.temporaryLocation" />}
-                        value={get(temporaryHoldingsLocation, ['name'], '-')}
+                        value={checkIfElementIsEmpty(holdingLocation.temporaryLocation)}
                       />
                     </Col>
                   </Row>
                   <Row>
-                    <Col smOffset={0} sm={4}>
+                    <Col
+                      smOffset={0}
+                      sm={4}
+                    >
                       <strong>
                         <FormattedMessage id="ui-inventory.itemLocation" />
                       </strong>
                     </Col>
                   </Row>
                   <br />
                   <Row>
-                    <Col smOffset={0} sm={4}>
+                    <Col
+                      smOffset={0}
+                      sm={4}
+                    >
                       <KeyValue
                         label={<FormattedMessage id="ui-inventory.permanentLocation" />}
-                        value={get(item, ['permanentLocation', 'name'], '-')}
+                        value={checkIfElementIsEmpty(itemLocation.permanentLocation)}
                       />
                     </Col>
                     <Col sm={4}>
                       <KeyValue
                         label={<FormattedMessage id="ui-inventory.temporaryLocation" />}
-                        value={get(item, ['temporaryLocation', 'name'], '-')}
+                        value={checkIfElementIsEmpty(itemLocation.temporaryLocation)}
                       />
                     </Col>
                   </Row>
                   <Row>
-                    <Col smOffset={0} sm={4}>
+                    <Col
+                      smOffset={0}
+                      sm={4}
+                    >
                       <strong>
                         <FormattedMessage id="ui-inventory.effectiveLocation" />
                       </strong>
                     </Col>
                   </Row>
                   <br />
                   <Row>
-                    <Col smOffset={0} sm={4}>
-                      {get(item, ['effectiveLocation', 'name'], '')}
+                    <Col
+                      smOffset={0}
+                      sm={4}
+                    >
+                      {checkIfElementIsEmpty(itemLocation.effectiveLocation)}
```

Comment:
This is just title, not field with value

Reply:
Are you sure? I see that the title is placed on line 1223.

Reply:
Ops, sorry, I looked to another line, now it's LGFM.